packages/atomic/cypress/integration/result-list/result-components-v1/result-date.cypress.ts

```diff
@@ -0,0 +1,121 @@
+import {generateComponentHTML} from '../../../fixtures/test-fixture';
+import {
+  interceptSearchResponse,
+  setUpPage,
+} from '../../../utils/setupComponent';
+import {executeFirstSearch} from '../../search-interface-utils.cypress';
+import {
+  generateResultList,
+  generateResultTemplate,
+  getFirstResult,
+} from '../result-list-v1-selectors';
+import {ResultDateSelectors} from './result-date-selectors';
+
+describe('Result Date Component', () => {
+  function setupResultDatePage(
+    props: Record<string, string | number>,
+    executeSearch = true
+  ) {
+    setUpPage(
+      generateResultList(
+        generateResultTemplate({
+          bottomMetadata: generateComponentHTML(
+            ResultDateSelectors.component,
+            props
+          ).outerHTML,
+        })
+      ),
+      executeSearch
+    );
+  }
+
+  function getFirstResultDate() {
+    return getFirstResult().find(ResultDateSelectors.component);
+  }
+
+  describe('when not used inside a result template', () => {
+    beforeEach(() => {
+      setUpPage(generateComponentHTML(ResultDateSelectors.component).outerHTML);
+    });
+
+    it.skip('should remove the component from the DOM', () => {
```

Comment:
This same test is present in other components and doesn't pass.

I personally prefer displaying the atomic-component-error instead of removing the component. What do you guys think?